ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/TableSuppress.java

```diff
@@ -57,6 +63,14 @@ public RefinementInfo getRefinementInfo() {
     return refinementInfo;
   }
 
+  public Formats getInternalFormats() {
+    return internalFormats;
+  }
+
+  public KsqlWindowExpression getWindowExpression() {
+    return windowExpression;
+  }
```

Comment:
is this used anywhere?

Reply:
I don't see it being called anywhere explicitly but when I remove it, I fail all my suppress qtts with the same error, 
`java.lang.RuntimeException: com.fasterxml.jackson.databind.exc.MismatchedInputException: Missing required creator property 'windowExpression' (index 4)`

Reply:
you need to also remove it from the constructor and the `@JsonProperty` annotation